synapse/storage/schema/main/delta/74/01_delete_e2e_backup_keys_for_deactivated_users.sql

```diff
@@ -0,0 +1,17 @@
+/* Copyright 2023 The Matrix.org Foundation C.I.C
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+INSERT INTO background_updates (ordering, update_name, progress_json) VALUES
+  (7324, 'delete_e2e_backup_keys_for_deactivated_users', '{}');
```

Comment:
```suggestion
  (7325, 'delete_e2e_backup_keys_for_deactivated_users', '{}');
```
`7324` seems to already be used [here](https://github.com/matrix-org/synapse/blob/develop/synapse/storage/schema/main/delta/73/24_events_jump_to_date_index.sql#L17).

Reply:
This should generally match the file path/name, so probably `7402`? (And rename the file to `02...sql`)

Reply:
right I've updated the filename and ordering, thank you for catching that!